cdip_connector/core/cdip_settings.py

```diff
@@ -19,6 +19,9 @@
 CDIP_API_ENDPOINT = env.str('CDIP_API_ENDPOINT', None)
 CDIP_ADMIN_ENDPOINT = env.str('CDIP_ADMIN_ENDPOINT', None)
 
+CDIP_API_SSL_VERIFY = env.bool('CDIP_API_SSL_VERIFY', True)
+CDIP_ADMIN_SSL_VERIFY = env.bool('CDIP_ADMIN_SSL_VERIFY', True)
```

Comment:
These settings are available so I can override them when using the SDK with services holding self-signed certificates. (Ex. portal-127.0.0.1.nip.io).